modules/apps/layout/layout-content-page-editor-web/src/main/java/com/liferay/layout/content/page/editor/web/internal/info/search/InfoSearchClassMapperRegistryUtil.java

```diff
@@ -15,31 +15,30 @@
 package com.liferay.layout.content.page.editor.web.internal.info.search;
 
 import com.liferay.info.search.InfoSearchClassMapperRegistry;
-
-import org.osgi.service.component.annotations.Component;
-import org.osgi.service.component.annotations.Reference;
+import com.liferay.osgi.util.service.Snapshot;
 
 /**
  * @author Jürgen Kappler
  */
-@Component(service = {})
 public class InfoSearchClassMapperRegistryUtil {
 
 	public static String getClassName(String searchClassName) {
-		return _infoSearchClassMapperRegistry.getClassName(searchClassName);
-	}
+		InfoSearchClassMapperRegistry infoSearchClassMapperRegistry =
```

Comment:
@JanisZhang These 3 `InfoSearchClassMapperRegistryUtil` seem to be completely same? If so, could you make it a public API to avoid repeating the same logic?